src/js/setup-text-tracks.js

```diff
@@ -30,6 +30,7 @@ function attachDashTextTracksToVideojs(player, tech, tracks) {
         label: track.lang,
```

Comment:
just noticed but should this be `track.label`?

Reply:
It would kind of make sense but the `track` here is  a [TextTrackInfo](https://github.com/Dash-Industry-Forum/dash.js/blob/master/src/streaming/vo/TextTrackInfo.js) and here's an example of what the properties look like:

```
kind: "captions"
label: 3
lang: "en"
roles: ["main"]
```

So it's possible that `track.label` was used here to show a user-friendly label

Reply:
Ah, I see. yeah, lang would be a better choice then.